docs/spec.md

```diff
@@ -5453,24 +5457,28 @@ The behavior of an infinite loop is TBD
 #### Examples
 
 ```mlir
-// %constant0: 1
-// %operand0: 0
-// %operand1: 10
-%results0, %results1 = "stablehlo.while"(%operand0, %operand1) ({
-  ^bb0(%arg0: tensor<i32>, %arg1: tensor<i32>):
-    %0 = "stablehlo.compare"(%arg0, %arg1) {
+// %init_i: 1
+// %init_sum: 0
+// %one: 1
+// %ten: 10
+%results0, %results1 = "stablehlo.while"(%init_i, %init_sum) ({
+  ^bb0(%curr_i: tensor<i64>, %curr_sum: tensor<i64>):
+    %cond = "stablehlo.compare"(%curr_i, %ten) {
```

Comment:
We have been consistently using  `%arg0, %arg1` for the arguments. Can we use that here as well?